docs/topics/proc_deploying-the-crud-booster-using-the-oc-cli-client.adoc

```diff
@@ -31,6 +31,8 @@ $ oc new-project {project-name}
 . Navigate to the root directory of your booster.
 
 . Deploy the PostgreSQL database to OpenShift.
+Ensure that you use the following login credentials when creating the database application.
+Using different credentials might prevent your booster application from integrating with the database.
```

Comment:
"Using different values will prevent your booster application from connecting to the database."

Reply:
Maybe also add one more line saying something like "This is because the booster application is preconfigured to use these values." It would be possible to also list the files where each booster uses these values (I provided the list in #926).

Reply:
Thank you:) IMHO that's definitely useful. I'll add that information in.